Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -480,6 +480,30 @@ mode would look as follows:
 
 .. _XDP acceleration:
 
+Socket LoadBalancer Bypass in Pod Namespace
+*******************************************
+
+Cilium has built-in support for bypassing the socket-level LoadBalancer and falling back
+to the tc LoadBalancer when a custom redirection/operation relies on the original ClusterIP
```

Comment:
What is a `tc LoadBalancer`? Maybe clarify here if easily possible.

Reply:
Added "at the veth interface" to help clarify, PTAL :)